server/src/main/java/com/github/animelist/animelist/model/input/profilepage/UserListBlockInput.java

```diff
@@ -0,0 +1,5 @@
+package com.github.animelist.animelist.model.input.profilepage;
+
+import org.bson.types.ObjectId;
+
+public record UserListBlockInput(ObjectId listId, int maxEntries) {}
```

Comment:
Is `listId` suppose to be ObjectId or String? I'm assuming requests will be passing in a String listId